mllib-common/src/test/scala/org/apache/spark/ml/attribute/AttributeGroupSuite.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ml.attribute
```

Comment:
Is this file exactly the same with master code file mllib/src/test/scala/org/apache/spark/ml/attribute/AttributeGroupSuite.scala ?

Reply:
Q2: Do we have other suites that need to be moved to `mllib-common` test directory ?

Reply:
They are the same.

Q2: Some suites are not copied to `mllib-common` due to cyclical dependency, I will list them in the PR description